File: woohyunkim/sort/H_Index.java
import java.util.Arrays;
class H_Index{
    public int solution(int[] citations) {
        int answer = 0;
        Arrays.sort(citations);
        for(int i=0; i<citations.length; i++){
           if(citations[i]==(citations.length-i)){
               return citations.length-i;
           }
        }
        return answer;
    }
}

File: donggyun/sort/H-Index.js
function solution(citations) {
  citations.sort((a, b) => b - a);
  for (let i = 0; i < citations.length; i++) {
    if (citations[i] === i + 1) return i + 1;
    else if (citations[i] < i + 1) return i;
  }
  return citations.length;
}
